cob_description/urdf/drive_wheel/drive_wheel.urdf.xacro

```diff
@@ -113,8 +113,16 @@
 
   <xacro:macro name="drive_wheel" params="*origin parent suffix drive_vel:=^|100 steer_vel:=^|100 passive:=false">
 
-    <xacro:cob_caster_hub parent="${parent}" suffix="${suffix}_caster" passive="${passive}" >
-      <xacro:insert_block name="origin" />
+    <link name="${suffix}_link"/>
+
+    <joint name="${suffix}_joint" type="fixed">
```

Comment:
why do you need this additional fixed joint?

Reply:
I still don't see a reason why we would need this joint and link
please remove again